assets/js/index.js

```diff
@@ -0,0 +1,914 @@
+	},
+
+	// Scroll to anchor
+	scrollToAnchor: function () {
+		var anchorElements = document.querySelectorAll('a[href*="#"]');
+		var anchorElementsList = Array.prototype.slice.call(anchorElements);
+		anchorElementsList.filter(function (element) {
+			if (element.href === '#' || element.href === '#0' || element.classList.contains('.do-not-scroll') || element.classList.contains('skip-link')) {
+				return false;
+			}
+			return true;
+		}).forEach(function (element) {
+			element.addEventListener('click', function (event) {
+				// On-page links
+				if (window.location.hostname === event.target.hostname) {
+					// Figure out element to scroll to
+					var target = window.location.hash !== '' && document.querySelector(window.location.hash);
+					target = target ? target : event.target.hash !== '' && document.querySelector('#' + event.target.hash);
+
+					// Does a scroll target exist?
+					if (target) {
+						// Only prevent default if animation is actually gonna happen
+						event.preventDefault();
+
+						// Get options
+						var additionalOffset = event.target.dataset.additionalOffset;
+						var scrollSpeed = event.target.dataset.scrollSpeed ? event.target.dataset.scrollSpeed : 500;
+
+						// Determine offset
+
+						var adminBar = document.querySelector('#wpadminbar');
+
+						var originalOffset = target.getBoundingClientRect().top + window.pageYOffset;
+						var scrollOffset = additionalOffset ? originalOffset + additionalOffset : originalOffset;
+						if (adminBar && event.target.className === 'to-the-top') {
+							scrollOffset = scrollOffset - adminBar.getBoundingClientRect().height;
+						}
+
+						twentytwentyScrollTo(scrollOffset, null, scrollSpeed);
+
+						window.location.hash = event.target.hash.slice(1);
+					}
+				}
+			});
+		});
+	},
+
+	// Scroll to element
+	scrollToElement: function () {
+		var scrollToElement = document.querySelector('*[data-scroll-to]');
+
+		if (scrollToElement) {
+			scrollToElement.addEventListener('click', function (event) {
+				// Figure out element to scroll to
+				var target = event.target.dataset.twentytwentyScrollTo;
+
+				// Make sure said element exists
+				if (target) {
+					event.preventDefault();
+
+					// Get options
+					var additionalOffset = event.target.dataset.additionalOffset,
+						scrollSpeed = event.target.dataset.scrollSpeed ? event.target.dataset.scrollSpeed : 500;
+
+					// Determine offset
+					var originalOffset = target.getBoundingClientRect().top + window.pageYOffset,
+						scrollOffset = additionalOffset ? originalOffset + additionalOffset : originalOffset;
+
+					twentytwentyScrollTo(scrollOffset, null, scrollSpeed);
+				}
+			});
+		}
+	},
+
+}; // twentytwenty.smoothScroll
+
+/*	-----------------------------------------------------------------------------------------------
+	Interval Scroll
+--------------------------------------------------------------------------------------------------- */
+
+twentytwenty.intervalScroll = {
```

Comment:
I think we can get rid of this function, the event isn't used anywhere...

Reply:
same for `twentytwenty.resizeEnd`

Reply:
agreed  👍 this far I only mirrored all the functionality from the original file. I did not jet take unused stuff out. Wasn't sure wether that should happen as part of the same PR.